aiocache/decorators.py

```diff
@@ -55,7 +55,9 @@ def cached_decorator(func):
 
             try:
                 if await cache_instance.exists(cache_key):
-                    return await cache_instance.get(cache_key)
+                    result = await cache_instance.get(cache_key)
+                    await cache_instance.close()
```

Comment:
Not entirely, I guess we will agree to disagree haha.  I have made the changes to use ensure future anyways.